andela/ah-the-jedi-backend
# #165305277 Implement estimated reading time for an article

### What does this PR do?
Show the estimated reading time for an article
### Description of Task(s) to be completed?
- ensure every article has an estimate read time
- write unit tests for the functionality

### How should this be manually tested?

Clone this repository on local machine and CD into project folder
`$ git clone git@github.com:andela/ah-the-jedi-backend.git`
`$ cd ah-the-jedi-backend`

Check out to the PR's branch
`$ git checkout ft-article-readtime-165305277`

Create and activate virtual environment
`$ python3 -m venv venv`
`$ source .venv/bin/activate`

Install app dependencies from requirements.txt
`$ pip install -r api/requirements.txt`

Make migrations and run the server
`$ python manage.py makemigrations`
`$ python manage.py migrate`
`$ python manage.py runserver`

**Use postman to test the endpoints**


           POST <your-localhost>/api/users/



       POST <your-localhost>/api/login/ 

       Use the token provided after login for authorization 

- [x] **Create an article**

      `POST`  `<you-local-host>/api/articles/`

- [x] **Get article**

      `GET`  `<you-local-host>/api/articles/<slug>/`


### What are the relevant PT stories?
[#165305277](https://www.pivotaltracker.com/story/show/165305277)

### Screenshots

**Payload showing article readtime**
<img width="1140" alt="Screenshot 2019-05-07 at 19 22 52" src="https://user-images.githubusercontent.com/45317755/57316037-9330da80-70fd-11e9-806e-c7db1ae2444d.png">
